modules/Kconfig.infineon

```diff
@@ -60,4 +60,9 @@ config HAS_XMCLIB_ETH
 	help
 	  Enable XMCLIB Ethernet MAC
 
+config HAS_XMCLIB_CAN
+	bool
+	help
+	  Enable XMCLIB CAN
```

Comment:
Where is this used?

Reply:
It's used to conditionally compile xmc_can.c from hal_infineon:
https://github.com/zephyrproject-rtos/hal_infineon/blob/master/XMCLib/drivers/CMakeLists.txt#L20